OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -142,4 +142,4 @@ public ConnectionFailedLogic(Widget widget, OrderManager orderManager, Action on
 			}
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
Actually, here it's adding an end-of-line newline (which didn't exist). I assume you want this, or? (since both git in diffs and github usually complain/warn about that).

The other two are actually two new lines at end of file (instead of just one), that my editor automatically removed. I initially kept them cause why not, but I'll get rid of them.